django/core/files/utils.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, file):
     readline = property(lambda self: self.file.readline)
     readlines = property(lambda self: self.file.readlines)
     seek = property(lambda self: self.file.seek)
+    seekable = property(lambda self: self.file.seekable)
```

Comment:
I can be explicit about it adding the property if that is preferred.  That was my original intention, but my test showed that the attribute wasn't added when you are in py2 so it seemed cleaner to leave it that way.  Let me know your preference, and I'll change it if requested